File: app/src/main/java/org/mifos/mobile/ui/views/UserDetailsView.java
package org.mifos.mobile.ui.views;

import android.graphics.Bitmap;

import org.mifos.mobile.models.client.Client;
import org.mifos.mobile.ui.views.base.MVPView;

/**
 * Created by naman on 07/04/17.
 */

public interface UserDetailsView extends MVPView {

    void showUserDetails(Client client);

    void showUserImage(Bitmap bitmap);

    void showError(String  message);
}

File: app/src/main/java/org/mifos/mobile/ui/views/UserDetailsView.kt
package org.mifos.mobile.ui.views

import android.graphics.Bitmap
import org.mifos.mobile.models.client.Client
import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by naman on 07/04/17.
 */
interface UserDetailsView : MVPView {
    fun showUserDetails(client: Client?)
    fun showUserImage(bitmap: Bitmap?)
    fun showError(message: String?)
}